src/Microsoft.Azure.WebJobs.Host/Extensions/ExtensionInfo.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.Azure.WebJobs
 {
-    public class ExtensionInfo
+    public class ExtensionInfo : IExtensionInfo
```

Comment:
We don't need the IExtensionInfo interface - ExtensionInfo is already a public type. Why can't you use that type in the interface above?

Reply:
Testability. The other solution is introduce a class that implements `IExtensionConfigProvider` on the TestClass. If you prefer the approach I will do it.

Reply:
We don't want to add new unnecessary public surface area just for our testability. To solve your test issue, just use one of the ExtensionInfo factory methods to create an instance.